.nvmrc

```diff
@@ -0,0 +1 @@
+0.10
```

Comment:
IMO it doesn't make much sense in here. This utility has to work with all node versions.